### PR TITLE
Language review of Edge Documentation

### PR DESCRIPTION
**Pull request by dariavladykina**
This PR aims to check the compliance of the Edge documentation with the SUSE Documentation Style Guide in terms of style, consistency and terminology.

### CODE REVIEW

File: asciidoc/quickstart/elemental.adoc [hunk 1]
```diff
@@ -168,35 +168,35 @@ helm install --create-namespace -n cattle-elemental-system \
  oci://registry.suse.com/rancher/elemental-operator-chart
 ----
 
-==== (Optionally) Install the Elemental UI Extension
+==== (Optionally) Install the Elemental UI extension
 
-. If you want to use the Elemental UI, you can log in to your Rancher instance, click on the "hamburger menu" in the top left, then
+. To use the Elemental UI, log in to your Rancher instance, click on the three-dot menu in the upper left, then
```
Comment (dariavladykina) [inline asciidoc/quickstart/elemental.adoc:173]: Some text seems to be missing after "then".

Comment (agracey) [reply asciidoc/quickstart/elemental.adoc:173]: Yeah, should finish with `select "Extensions"`

File: asciidoc/quickstart/elemental.adoc [hunk 2]
```diff
@@ -324,7 +324,7 @@ EOF
 [NOTE]
 ====
 * The unencoded password is `eib`
-* The `cat` command has `\` before each `$` so that bash doesn't template them. If you want to copy the encoded password separately it is `$6$jHugJNNd3HElGsUZ$eodjVe4te5ps44SVcWshdfWizrP.xAyd71CVEXazBJ/.v799/WRCBXxfYmunlBO2yp1hm/zb4r8EmnrrNCF.P/`.
+* The `cat` command has `\` before each `$` so that Bash does not template them. If you want to copy the encoded password separately, it is `$6$jHugJNNd3HElGsUZ$eodjVe4te5ps44SVcWshdfWizrP.xAyd71CVEXazBJ/.v799/WRCBXxfYmunlBO2yp1hm/zb4r8EmnrrNCF.P/`.
```
Comment (dariavladykina) [inline asciidoc/quickstart/elemental.adoc:327]: This is a style sheet issue - the password is rendered with no breaks in the output, overlapping with the right-hand menu.

File: asciidoc/quickstart/elemental.adoc (hunk 1 above)
Comment (agracey) [reply asciidoc/quickstart/elemental.adoc:173]: I'm tempted to say that we should actually drop the UI portion of the quick start as I think it likely causes more confusion (due to several UI elements that break when using EIB to build installation media)

File: asciidoc/quickstart/elemental.adoc (hunk 2 above)
Comment (agracey) [reply asciidoc/quickstart/elemental.adoc:327]: At least on my browser, it does seem to give a horizontal scroll bar to see the rest

File: asciidoc/quickstart/eib.adoc [hunk 3]
```diff
@@ -11,9 +11,9 @@ ifdef::env-github[]
 :warning-caption: :warning:
 endif::[]
 
-Edge Image Builder (EIB) is a tool that streamlines the process of generating pre-configured, ready-to-deploy (RTD) disk images for bootstrapping machines, even in air-gapped scenarios.
+Edge Image Builder (EIB) is a tool that streamlines the process of generating preconfigured, ready-to-deploy (RTD) disk images for bootstrapping machines, even in air-gapped scenarios.
 
-For more information, read the <<components-eib,Edge Image Builder Introduction>>
+For more information, read the <<components-eib,Edge Image Builder Introduction>>.
 
 == Prerequisites
```
Comment (dariavladykina) [inline asciidoc/quickstart/eib.adoc:18]: This section is orphaned as a single top-level item. I believe the entire structure must be updated to make several same-level chapters (e.g. Prerequisites, Getting the EIB image, Creating the image configuration directory, and so on) with their respective subordinate sections.

File: asciidoc/quickstart/eib.adoc [hunk 4]
```diff
@@ -23,7 +23,7 @@ For more information, read the <<components-eib,Edge Image Builder Introduction>
 
 [NOTE]
 ====
-If you are running Rancher Desktop, you need to switch to the https://docs.rancherdesktop.io/ui/preferences/container-engine/general[`dockerd (moby)` container runtime]
+If you are running Rancher Desktop, you need to switch to the https://docs.rancherdesktop.io/ui/preferences/container-engine/general[`dockerd (moby)` container runtime].
 ====
 
 === Getting the EIB Image
```
Comment (dariavladykina) [inline asciidoc/quickstart/eib.adoc:29]: There is no content in this section, only the screen. Please add some introduction for it.

File: asciidoc/quickstart/elemental.adoc (hunk 2 above)
Comment (dariavladykina) [reply asciidoc/quickstart/elemental.adoc:327]: In other occurrences, I get the scroll bar for such long strings, too. This is a one-off for some reason.

File: asciidoc/quickstart/eib.adoc (hunk 4 above)
Comment (agracey) [reply asciidoc/quickstart/eib.adoc:29]: What do you mean?

Comment (dariavladykina) [reply asciidoc/quickstart/eib.adoc:29]: I see the output like this:

image

Comment (dariavladykina) [reply asciidoc/quickstart/eib.adoc:29]: I see the output like this: 

image

Comment (agracey) [reply asciidoc/quickstart/eib.adoc:29]: Oh... I read your comment wrong and was thinking it was about the Rancher Desktop link. This makes way more sense 😄

File: asciidoc/quickstart/eib.adoc (hunk 3 above)
Comment (agracey) [reply asciidoc/quickstart/eib.adoc:18]: Agreed. I think this is a remnant of the markdown version

**Comment by agracey** [comment]
@dariavladykina Would you prefer me to make fixes in this PR or go ahead and merge this then make fixes in my own branch?

**Comment by dariavladykina** [comment]
Hi @agracey, I believe it's better to make fixes in this PR and then re-request another final eye review. Please bear with me to finish all the files so that we don't get validation errors. Thank you!

**Review by dariavladykina — COMMENTED** [review]
Added a few comments about broken links that I spotted.

**Review by fsundermeyer — APPROVED** [review]
LGTM, ready to be merged. Please turn the remaining commenmts into issues

**Pull request merged**